src/components/App.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+/* eslint-enable no-unused-vars */
+
+function App() {
+  return (
+      <React.Fragment>
+        <Display result ='Milestone 2 (No calculation till now)'/>
+        <ButtonPanel />
+      </React.Fragment>
+  );
+}
```

Comment:
- [ ] Kindly use es6 function syntax here. Fix this and other similar functions.

Reply:
okay